Add vignettes

csoneson opened the pull request.

csoneson committed:
Add 'How to compute read coverage' vignette

csoneson committed:
Shorten some lines, remove unnecessary spaces

csoneson committed:
Add vignette for extracting exons and intron sequences for a gene

csoneson committed:
Add more dependencies

csoneson committed:
Expand .Rbuildignore

csoneson committed:
Add Rsamtools dependency

mbstadler committed:
update URL in _pkgdown.yml

mbstadler committed:
explicitely list navbar components to suppress empty "Reference"

csoneson committed:
Merge

Merge branch 'devel' into add-vignettes-cs-202504

# Conflicts:
#	DESCRIPTION
#	_pkgdown.yml

mbstadler reviewed (approved):
This all looks very good to me! I guess the check failures are all related to missing macOS binaries...

csoneson committed:
Add Windows to GHA

mbstadler committed:
add URL to DESCRIPTION

mbstadler committed:
add author to DESCRIPTION

mbstadler committed:
update quarto template to always embed-resources

mbstadler committed:
add new vignette on how to compute sequence composition for genomic regions

mbstadler committed:
minor edits to sequence-composition vignette

mbstadler committed:
add link to GitHub repo in DESCRIPTION to get a link in pkgdown

mbstadler committed:
use compact list representation in _pkgdown.yml

mbstadler committed:
fix conflicts

Merge remote-tracking branch 'origin/devel' into add-vignettes-mbs-20250423

# Conflicts:
#	DESCRIPTION
#	_pkgdown.yml

mbstadler committed:
fix conflicts

Merge remote-tracking branch 'origin/devel' into add-vignettes-mbs-20250423

# Conflicts:
#	DESCRIPTION
#	_pkgdown.yml

mbstadler committed:
Merge branch 'add-vignettes-mbs-20250423' of github.com:mbstadler/BiocHowTo into add-vignettes-mbs-20250423

csoneson committed:
Merge pull request #11 from mbstadler/add-vignettes-mbs-20250423

Add vignette on how to compute sequence features for given regions

csoneson committed:
Add Bioc styling to pkgdown page

csoneson commented:
Merging after offline discussion and approval from @almeidasilvaf
